gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/NonLocalRegionEntry.java

```diff
@@ -46,7 +47,7 @@
 import com.gemstone.gemfire.internal.offheap.annotations.Released;
 import com.gemstone.gemfire.internal.shared.Version;
 
-public class NonLocalRegionEntry implements RegionEntry, VersionStamp {
+public class NonLocalRegionEntry implements RegionEntry, VersionStamp, DataSerializable {
```

Comment:
Looks like, as discussed, serialization of this entry not required as this will never be persisted or sent over the wire.